### PR TITLE
Upgrade object manipulation

### PR DESCRIPTION
**Pull request by saxtot**
Object manipulation now includes:

- Removing a single object.
- Removing all objects.
- Adding Mesh type objects (needs a matching version of `curobo_msgs`).
- Checking to confirm the validity of added objects. For example, existing objects or those that would be under the ground are denied.

Other changes:

- README now has a section for object manipulation, with examples.
- The `publish_markers_voxel` function now also deletes old voxels, allowing RViz to stay up to date. The file was also formatted.
- Optimized `world_cfg` management, it is now only created once and then updated continuously.
- The panel for adding objects is in the RViz configuration file.

### CODE REVIEW

**Review by will-44 — COMMENTED** [review]
L'ajout de mesh ne fonctionne pas de mon cote que ce soit avec un stl ou un dae. Mettre un tuto pour tester cette fonctionnalité facilement

File: README.md [hunk 1]
```diff
@@ -104,6 +104,38 @@ ros2 run tf2_ros static_transform_publisher  0.5 0 0.5 0 0 0  base_0 camera_link
 
 Remplacer ces champs par ce qui vous convient.
 
+### Gestion manuelle d'objets virtuels
+
+Des services ont été ajoutés pour ajouter et retirer différents types d'objets à l'environnement.
+
+Présentement, il est possible d'ajouter ces types:
```
[the remark this answers is not in the document]
Comment (saxtot) [reply README.md:111]: Fait!

File: README.md [hunk 2]
```diff
@@ -104,6 +104,38 @@ ros2 run tf2_ros static_transform_publisher  0.5 0 0.5 0 0 0  base_0 camera_link
...
+
+- Cube (Cuboid selon l'appellation cuRobo)
+- Sphère
+- Capsule
+- Cylindre
+- Mesh à partir d'un fichier fournis par le paramètre optionnel `mesh_file_path`
```
[the remark this answers is not in the document]
Comment (saxtot) [reply README.md:117]: C'est ajouté plus bas.

File: curobo_ros/core/config_wrapper.py [hunk 3]
```diff
@@ -145,16 +187,104 @@ def callback_add_object(self, node, request: AddObject, response):
...
             self.world_cfg.add_obstacle(obstacle)
-            node.motion_gen.world_coll_checker.clear_cache()
-            node.motion_gen.update_world(self.world_cfg)
+            self.update_world_config(node)
             response.message = 'Object ' + request.name + ' added successfully'
 
         return response
 
-    #TODO remove object from name
+    def callback_remove_object(self, node, request: RemoveObject, response):
```
[the remark this answers is not in the document]
Comment (saxtot) [reply curobo_ros/core/config_wrapper.py:238]: C'est fait. J'ai aussi ajouté un log pour confirmer le succès d'un ajout.

**Review by will-44 — APPROVED** [review]

**Pull request merged**